### DISCUSSION TITLE
Character encoding error

### DISCUSSION THREAD

**Discussion by bentodvictor** [docsifyjs/docsify#1829] [Q&A] [answered]
Hello everyone.

I'm try to use Docsify.js with ASP.NET MVC 5 and I founded problems in charactere enconding.

* This is the folder structure:
![image](https://user-images.githubusercontent.com/32740824/176891798-7e24d9e8-2e17-4798-8bce-72c2c04b4c80.png)

* This is my cshtml file:
```
@{
    Layout = null;
}

<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width,initial-scale=1">

    <title>Docsify Test</title>

    <link rel="stylesheet" href="~/assets/css/docsify/vue.css" />

    <style>
        .app-name {
            margin-bottom: 2rem !important;
        }
    </style>

    <script defer src="~/assets/js/scripts/docsify/docsify.min.js"></script>
    <script defer src="~/assets/js/scripts/docsify/search.min.js"></script>
    <script defer src="~/assets/js/scripts/docsify/docsify-pagination.min.js"></script>
    <script defer src="~/assets/js/scripts/docsify/emoji.min.js"></script>
</head>
<body>
    <div data-app id="main"></div>
    <script>

        // Set html "lang" attribute based on URL
        var lang = location.hash.match(/#\/(br|es)\//);
        if (lang) {
            document.documentElement.setAttribute('lang', lang[1]);
        }

        window.$docsify = {
            el: '#main',
            basePath: '/docs/',
            loadNavbar: true,
            loadSidebar: true,
            auto2top: true,
            mergeNavbar: true,
            search: {
                paths: 'auto',
                placeholder: {
                    '/br/': 'Pesquisar',
                    '/es/': 'Buscar',
                    '/': 'Search'
                },
                noData: {
                    '/br/': 'Sem resultados',
                    '/es/': 'No hay resultados',
                    '/': 'No results'
                },
                pathNamespaces: ['/es', '/br']
            },

            name: 'Docsify Test',
            themeColor: '#2a5c9a',
        }
    </script>

</body>
</html>
```

* Differences between English and Spanish translation:
![uslanguage](https://user-images.githubusercontent.com/32740824/176892271-70cb849d-75ed-4306-96f9-7795d9a1f0ce.png)
(Soluci**ó**n)
![eslanguage](https://user-images.githubusercontent.com/32740824/176892285-c1cfcd2c-9952-4f9e-a425-e2318bcbdcda.png)

Has anyone had this same issue?
Thanks in advance.

**Comment by sy-records** [comment]
This doesn't seem to be related to docsify, it might be a browser coding issue.

**Reply by bentodvictor** [reply]
Thanks for your reply.

To discart this possibility I added a `h5` tag with some special characteres inside.
![Screenshot_2](https://user-images.githubusercontent.com/32740824/177363559-f01c618a-facf-49fa-90a5-ad64b077b432.png)

Also, I added the same content of the `h5` to the `.md` file.
The `h5` tag rendered correctly but the content of  `.md` file not.
![Screenshot_1](https://user-images.githubusercontent.com/32740824/177363605-e654dc94-d71b-493a-9a42-128b2657f4b3.png)

**Reply by sy-records** [reply]
Can you provide a reproducible repo?

**Reply by bentodvictor** [reply]
I found a "solution". I simple deleted the default file and re-create it, and all works.
Thank You!!!

**Comment by bentodvictor** [comment, marked as answer]
I found a "solution". I simple deleted the default file and re-create it, and all works.
